webpack/scenes/ContentViews/Create/CreateContentViewFormComponents.js

```diff
@@ -0,0 +1,93 @@
+import { translate as __ } from 'foremanReact/common/I18n';
+import React from 'react';
+import { Popover, Flex, FlexItem } from '@patternfly/react-core';
+import { HelpIcon } from '@patternfly/react-icons';
+import ContentViewIcon from '../components/ContentViewIcon';
+
+
+export const LabelComposite = () => (
+  <Flex>
+    <FlexItem spacer={{ default: 'spacerNone' }}><ContentViewIcon composite /></FlexItem>
+    <FlexItem>Composite Content View</FlexItem>
+  </Flex>
+);
+
+export const LabelComponent = () => (
+  <Flex>
+    <FlexItem spacer={{ default: 'spacerNone' }}><ContentViewIcon composite={false} /></FlexItem>
+    <FlexItem>Component Content View</FlexItem>
+  </Flex>
+);
+
+export const LabelDependencies = () => (
+  <Flex>
+    <FlexItem spacer={{ default: 'spacerSm' }}>Solve Dependencies </FlexItem>
+    <FlexItem>
+      <Popover
+        bodyContent={
+            __('This will solve RPM and Module Stream dependencies on every publish of this Content View. ' +
+            'Dependency solving significantly increases publish time (publishes can take over three times as long) ' +
+            'and filters will be ignored when adding packages to solve dependencies. ' +
+            'Also, certain scenarios involving errata may still cause dependency errors.')
+          }
+        aria-label="Solve Dependencies"
+        position="right"
+      >
+        <button
+          aria-label="More info for dependenices field"
+          onClick={e => e.preventDefault()}
+          className="pf-c-form__group-label-help"
+        >
+          <HelpIcon noVerticalAlign />
+        </button>
+      </Popover>
+    </FlexItem>
+  </Flex>
+);
+
+export const LabelAutoPublish = () => (
+  <Flex>
+    <FlexItem spacer={{ default: 'spacerSm' }}>Auto Publish </FlexItem>
+    <FlexItem>
+      <Popover
+        bodyContent={
+            __('Automatically publish a new version of the composite content view whenever one of its component content views is published. ' +
+          '(Also note auto-publish will only happen for component views that use the "Always use latest" option.)')
+          }
+        aria-label="Auto Publish"
```

Comment:
To be consistent with the new details page this should be
```suggestion
        aria-label="Autopublish"
```
(however, let's talk about that at our next meeting)